src/org/loklak/harvester/HackernewsRSSReader.java

```diff
@@ -0,0 +1,107 @@
+/**
+ *  Hacker News RSS Reader
+ *  By Jigyasa Grover, @jig08
+ **/
+
+package org.loklak.harvester;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.List;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import com.sun.syndication.feed.synd.SyndEntry;
+import com.sun.syndication.feed.synd.SyndFeed;
+import com.sun.syndication.io.SyndFeedInput;
+import com.sun.syndication.io.XmlReader;
```

Comment:
Added in https://github.com/loklak/loklak_server/pull/610/commits/488f17b1b0fb1bf8a2e2e56452e582a9ea6d3c0c